emperor/support_files/js/controller.js

```diff
@@ -209,6 +209,9 @@ define([
         controller.resize(w, h);
       }
     });
+
+    // Set for update 
+    this.sceneViews[0].needsUpdate = true;
```

Comment:
If any sceneview needs updating all are, so I just used 0 since it will always exist.